calibration/DOFSTK/CGINS-DOFSTK-02628__20190313.csv

```diff
@@ -0,0 +1,7 @@
+serial,name,value,notes
+43-2628,CC_frequency_offset,-8.2051e+002,
+43-2628,CC_oxygen_signal_slope,3.2368e-004,
+43-2628,CC_residual_temperature_correction_factor_a,-5.0501e-003,
+43-2628,CC_residual_temperature_correction_factor_b,1.9640e-004,
+43-2628,CC_residual_temperature_correction_factor_c,-3.1581e-006,
+43-2628,CC_residual_temperature_correction_factor_e,3.600000e-002,
```

Comment:
SNW reviewed - OK

Reply:
Reviewed by CDobs. Looks good. I see the correct value for CC_oxygen_signal_slope in the SBE 43F O2628 SOCAdjusted 13Mar19.pdf.